docs/source/projects.rst

```diff
@@ -302,6 +302,19 @@ Docker container environment
   When the MLflow project is run, Docker attempts to pull the image from the specified registry.
   The system executing the MLflow project must have credentials to pull this image from  the specified registry.
 
+  .. rubric:: Example 4: Using a prebuilt image
+
+  .. code-block:: yaml
+
+    docker_env:
+      image: python:3.7
+      rebuild: false
```

Comment:
@harupy The more I think about it, I think it makes more sense to structure this as an API / commandline argument for `mlflow run`. This way, the user can specify at runtime whether or not to rebuild the image, rather than having to change the project's contents.